src/ContainerDetails.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+class ContainerDetails extends React.Component {
+    constructor(props) {
+        super(props);
+    }
```

Comment:
This component has no state, so it should be a  [functional component](https://reactjs.org/docs/components-and-props.html#functional-and-class-components). ([another interesting blog post](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc)). Also, it's custom to explicitly spell out the properties that you support/expect, like this:
```js
const ContainerDetails({container}) => (
    <div> ...
        <dt>{ container.Created.. }</dt>
    </div>
);
```